### ISSUE TITLE
OCSPscrape config improvements

### ISSUE THREAD

**Issue by scolby33** [scolby33/OCSPdash#39]
- [ ] better loading of default host
- [ ] alternate options for loading a private key

Config files? Env variables?

**Comment by cthoyt** [comment]
Will maybe use https://github.com/mbr/flask-appconfig

**Comment by scolby33** [comment]
The shear number of flask-* things you add to this project is amazing.

**Comment by cthoyt** [comment]
Nope flask-appconfig is defunct

**Comment by cthoyt** [comment]
i changed my mind. Keyword arguments in the create_application functions or environment variables or bust

**Comment by scolby33** [comment]
I'm confused

Hey how about `def myfunction(kwarg1=whatever, magic_thing=whatever)` and some inspection of os.env to autoextract `f'{ENVPREFIX}_KWARG1'` and `f'{ENVPREFIX}_MAGIC_THING'`?

**Comment by cthoyt** [comment]
Yeah this is the idea of namespacing within envionrment variables. Click actually does this for you in their CLI

**Comment by scolby33** [comment]
It looks like it works basically the way I wrote up there except it uses `'%s_%s' % (...)` instead of f strings.

**Comment by cthoyt** [comment]
So is this done? I've decided never ever to use config files again until i find a library that basically does all of the searching for you (the same way that all of flake8, mypy, etc. are able to look in setup.cfg and also their specific files, but i don't want to write that code myself)

**Comment by scolby33** [comment]
We wrote most of that code for pushover_complete, I think.

You should write the library.

**Comment by cthoyt** [comment]
Okay. Maybe build on this https://github.com/poudel/simple-config

**Comment by cthoyt** [comment]
There’s this new repo called easy_config I think we should use
